src/sentry/web_vitals/issue_platform_adapter.py

```diff
@@ -0,0 +1,100 @@
+import hashlib
+from datetime import UTC, datetime
+from uuid import uuid4
+
+from sentry.issues.grouptype import WebVitalsGroup
+from sentry.issues.ingest import hash_fingerprint
+from sentry.issues.issue_occurrence import IssueEvidence, IssueOccurrence
+from sentry.issues.producer import PayloadType, produce_occurrence_to_kafka
+from sentry.models.group import Group, GroupStatus
+from sentry.web_vitals.types import WebVitalIssueDetectionType, WebVitalIssueGroupData
+
+
+def create_fingerprint(vital: WebVitalIssueDetectionType, transaction: str) -> str:
+    prehashed_fingerprint = f"insights-web-vitals-{vital}-{transaction}"
+    fingerprint = hashlib.sha1((prehashed_fingerprint).encode()).hexdigest()
+    return fingerprint
+
+
+def send_web_vitals_issue_to_platform(data: WebVitalIssueGroupData, trace_id: str) -> None:
+    # Do not create a new web vital issue if an open issue already exists
+    if check_unresolved_web_vitals_issue_exists(data):
+        return
+
+    event_id = uuid4().hex
+    now = datetime.now(UTC)
+    transaction = data["transaction"]
+    vital = data["vital"]
+
+    tags = {
+        "transaction": data["transaction"],
+        "web_vital": vital,
+        "score": f"{data['score']:.2g}",
+        vital: f"{data['value']}",
+    }
+
+    event_data = {
+        "event_id": event_id,
+        "project_id": data["project"].id,
+        "platform": data["project"].platform,
+        "timestamp": now.isoformat(),
+        "received": now.isoformat(),
+        "tags": tags,
+    }
+
+    if trace_id:
+        event_data["contexts"] = {
+            "trace": {
+                "trace_id": trace_id,
+                "type": "trace",
+            }
+        }
+
+    evidence_data = {
+        "transaction": transaction,
+    }
+
+    evidence_display = [
+        IssueEvidence(
+            name="Transaction",
+            value=transaction,
+            important=False,
+        ),
+    ]
+
+    # TODO: Add better titles and subtitles
+    title = f"{data['vital'].upper()} score needs improvement"
```

Comment:
I forgot, but can we add some kind of experimental label/flag/prefix to the title so people will know this is only experimental as we continue tweaking it?